config/toml.go

```diff
@@ -322,29 +325,17 @@ func ResetTestRoot(testName string) *Config {
 }
 
 func ResetTestRootWithChainID(testName string, chainID string) *Config {
-	rootDir := os.ExpandEnv("$HOME/.tendermint_test")
-	rootDir = filepath.Join(rootDir, testName)
-	// Remove ~/.tendermint_test_bak
-	if cmn.FileExists(rootDir + "_bak") {
-		if err := os.RemoveAll(rootDir + "_bak"); err != nil {
-			cmn.PanicSanity(err.Error())
-		}
-	}
-	// Move ~/.tendermint_test to ~/.tendermint_test_bak
-	if cmn.FileExists(rootDir) {
-		if err := os.Rename(rootDir, rootDir+"_bak"); err != nil {
-			cmn.PanicSanity(err.Error())
-		}
-	}
-	// Create new dir
-	if err := cmn.EnsureDir(rootDir, 0700); err != nil {
-		cmn.PanicSanity(err.Error())
+	// create a unique, concurrency-safe test directory under $HOME/.tendermint_test/
+	rootDir, err := ioutil.TempDir("", fmt.Sprintf("%s-%s_", chainID, testName))
```

Comment:
```suggestion
	rootDir, err := ioutil.TempDir(os.ExpandEnv("$HOME/.tendermint_test/"), fmt.Sprintf("%s-%s_", chainID, testName))
```

Reply:
From https://github.com/tendermint/tendermint/pull/3291/commits/ce1db7d579c0e97ff755a91ddb259c6ee6cf0544:

```
Place test directories in OS's tempdir

In modern UNIX and UNIX-like systems /tmp is very often
mounted as tmpfs. This might speed test execution a bit.
```

Any compelling reason why you want test dirs in your $HOME?

Reply:
I am not 100% sure, but we had some issues exactly because it's a different partition. But let's try /tmp again. Can you please change the comment 

```
// create a unique, concurrency-safe test directory under $HOME/.tendermint_test/
```
?

Reply:
Done, thanks!